tests/e2e-kubernetes/testsuites/credentials.go

```diff
@@ -86,6 +87,16 @@ type s3CSICredentialsTestSuite struct {
 	tsInfo storageframework.TestSuiteInfo
 }
 
+func determineServiceAccountTokenAudienceEKS() string {
```

Comment:
Lets follow up on removing the duplication here

Reply:
Agree, I will follow up next week.